docs.html

```diff
@@ -651,6 +652,135 @@ <h4>Card with Overlay</h4>
+              Email, etc.
+            </p>
+            <h4>Demo</h4>
+            <div class="well">
+              <div class="input-container">
+                <input type="text" placeholder="Placeholder Text" />
+                <span class="focus-border">
+                  <i></i>
+                </span>
+              </div>
+            </div>
+            <pre>
+              <code class="html">
+<div class="input-container">
+  <input type="text" placeholder="Placeholder Text" />
+  <span class="focus-border">
+    <i></i>
+  </span>
+</div>
+              </code> 
+            </pre>
+
+            <h4>Input with Validation</h4>
+            <div class="well">
+              <div class="input-container error">
+                <input type="email" placeholder="Email Address" />
+                <span class="input-error"> Please enter a valid email. </span>
+              </div>
+            </div>
+            <pre>
+              <code class="html">
+<div class="input-container error">
+  <input type="email" placeholder="Email Address" />
+  <span class="input-error"> Please enter a valid email. </span>
+</div>
+            </code>
+          </pre>
+          </section>
+
+          <section id="text">
+            <h1>Text</h1>
+            <p>There are various txt utilities available in Max UI</p>
+            <h4>Headings</h4>
+            <div class="well">
+              <h1>A quick brown fox jumps over the lazy dog</h1>
+              <h2>A quick brown fox jumps over the lazy dog</h2>
+              <h3>A quick brown fox jumps over the lazy dog</h3>
+              <h4>A quick brown fox jumps over the lazy dog</h4>
+              <h5>A quick brown fox jumps over the lazy dog</h5>
+              <h6>A quick brown fox jumps over the lazy dog</h6>
+            </div>
+            <pre>
+              <code class="html">
+<h1>A quick brown fox jumps over the lazy dog</h1>
+<h2>A quick brown fox jumps over the lazy dog</h2>
+<h3>A quick brown fox jumps over the lazy dog</h3>
+<h4>A quick brown fox jumps over the lazy dog</h4>
+<h5>A quick brown fox jumps over the lazy dog</h5>
+<h6>A quick brown fox jumps over the lazy dog</h6>
+              </code>
+            </pre>
+
+            <h4>Gray Text</h4>
+            <div class="well">
+              <p class="gray-4">A quick brown fox jumps over the lazy dog</p>
+              <p class="gray-3">A quick brown fox jumps over the lazy dog</p>
+              <p class="gray-2">A quick brown fox jumps over the lazy dog</p>
+              <p class="gray-1">A quick brown fox jumps over the lazy dog</p>
+            </div>
+            <pre>
+              <code class="html">
+<p class="gray-4">A quick brown fox jumps over the lazy dog</p>
+<p class="gray-3">A quick brown fox jumps over the lazy dog</p>
+<p class="gray-2">A quick brown fox jumps over the lazy dog</p>
+<p class="gray-1">A quick brown fox jumps over the lazy dog</p>
+              </code>
+            </pre>
+
```

Comment:
maintain code indentation

Reply:
Actually if I indent this code, it will come indented in the UI.